ext/Common.jl

```diff
@@ -156,3 +157,18 @@ end
 function inverse_target_transform(transformed_act_frac)
     return @. (1.0 / (2.0 * 0.99)) * tanh(transformed_act_frac) + 0.5
 end
+
+function calibration_error_metrics(X, Y, ensemble, aip, tps, FT)
+    N_ensemble = length(ensemble[1, :])
+    rmse = zeros(N_ensemble)
+    for i in 1:N_ensemble
+        pred = calibrated_prediction(X, ensemble[:, i], aip, tps, FT)
+        rmse[i] = StatsBase.mean(sqrt.((pred .- Y) .^ 2))
```

Comment:
If we only need the mean, would it be ok to compute it by ourselves? Instead of adding another dependency?